ngraph/frontend/onnx_import/src/utils/recurrent.cpp

```diff
@@ -37,7 +37,7 @@ namespace ngraph
             {
```

Comment:
As I understand `RecurrentSequence` class can be removed now?

Reply:
It's really nice piece of code, but you're right, we can remove it, in favour of the RNN/GRU Sequence classes

Reply:
Done